open-craft/xblock-group-project-v2
# Ta graded stages stats

Fixes https://edx-wiki.atlassian.net/browse/MCKIN-3684

**Testing instructions:**
1. Log in as TA, navigate to GP v2 admin dashboard
2. Note that Peer Grading stage in TA graded activity shows 100% incomplete stats (unless already reviewed as TA - than should display actual stats depending on actual review - see later).
3. Jump to detail view (ideally in a new window - you'll need the previous one later)
4. Jump to TA grading for one of the groups (simplest way - in detail view expand group, click on an icon that appears in the same cell to the right)
5. Provide grade.
6. Return to main and details view, refresh the page (in main you might need to select filters again)
7. Note that completion stats have changed.
7.1. In details view, all users in reviewed group should have "complete" state.
7.2. In main view, percent breakdown should be updated according to the number of students in the group and total number of students in all groups.
